src/main/core/Controller/APINew/Log/SecurityLogController.php

```diff
@@ -34,4 +44,26 @@ public function listAction(Request $request): JsonResponse
             []
         ));
     }
+
+    /**
+     * @Route("/list/current", name="apiv2_logs_security_list_current", methods={"GET"})
+     */
+    public function UserLogSecurtiyAction(Request $request): JsonResponse
```

Comment:
Typo. And methods should start with a lowercase

Reply:
Indeed, a small mistake sorry, thx :smile: